### ISSUE TITLE
Add MS MARCO document to Solr

### ISSUE THREAD

**Issue by lintool** [castorini/anserini#1074]
[Solrini](https://github.com/castorini/anserini/blob/master/docs/solrini.md) currently works only on MS MARCO passages. Would be nice to get it working on MS MARCO documents as well.

**Comment by lintool** [comment]
Closed by #1077

**Comment by lintool** [comment]
We seem to be getting minor differences:

```
2020-04-09 23:40:44,501 INFO - [FAILED] 0.2306 MAP, expected 0.231 MAP!
```

We should look into this...

**Comment by lintool** [comment]
Closed by #1115
